quickstart/www/analytics/student-reporting.php

```diff
@@ -0,0 +1,86 @@
+<?php
+    /**
+     * Copyright (c) 2017 Learnosity, MIT License
+     *
+     * Basic example of embedding student reports using Reports API.
+     */
+
+    // - - - - - - Section 1: server-side configuration - - - - - - //
+
+    // Include server side Learnosity SDK
+    require_once __DIR__ . '/../../src/vendor/autoload.php';
+    use LearnositySdk\Request\Init as LearnosityInit;
+
+    $request = [
+        // Array of report objects, one per report to render.
+        'reports' => [
+            // First report object.
+            [
+                'type'        => 'sessions-summary',    // Desired report to render.
+                'id'          => 'quickstart-report-1', // ID of the <span> in which to render.
+                'user_id'     => 'student_0001',        // Unique student identifier.
+                'session_ids' => ['ef4f80b8-e281-41f4-9efd-349b7eb9dd37'] // The session(s) to report on.
+            ],
+            // Second report object.
+            [
+                'type'        => 'sessions-summary-by-tag',
+                'id'          => 'quickstart-report-2',
+                'user_id'     => 'student_0001',
+                'session_ids' => [ 'ef4f80b8-e281-41f4-9efd-349b7eb9dd37' ],
+
+                // This report requires a preconfigured 'tag hierarchy', which is used to categorize
+                // the student's score into a drill down report of learning areas.
+                'hierarchy'   => 'quickstart_examples_hierarchy_001'
+            ],
+        ]
+    ];
+
+    // Public & private security keys required to access the Learnosity APIs and
+    // data. These keys grant access to Learnosity's public demos account.
+    // Learnosity will provide keys for your own private account.
+    $consumerKey = 'yis0TYCu7U9V4o7M';
+    $consumerSecret = '74c5fd430cf1242a527f6223aebd42d30464be22';
+
+    // Parameters used to create security authorization.
+    $security = [
+        'domain'       => $_SERVER['SERVER_NAME'],
+        'consumer_key' => $consumerKey
+    ];
+
+    // Use Learnosity SDK to construct Reports API configuration parameters,
+    // and sign them securely with the $security and $consumerSecret parameters.
+    $init = new LearnosityInit(
```

Comment:
```suggestion
    $init = new Init(
```

Reply:
Updated. I couldn't use your commit for some reason, so I replicated it manually.